docs/metadata_tables.md

```
@@ -303,88 +336,85 @@
+| arch | arm64 armhf x86-64 |
+| contributor | Tearran |
+| port | false |
+| extend_docs | false |
+| require_kernel | 5.15+ |
+| require_os | Debian Ubuntu Armbian |
+| maintainer | true |
+| unique_id | HEL013 |
+| description | Output Markdown table for module metadata arrays |
+| group | helpers |
+| extend_disc | Outputs a GitHub-flavored Markdown table for the provided associative array of metadata or options, grouping by section. Useful for automated documentation generation and readability. |
+| feature |  |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)